docs/en/client/java-sdk-user-guide.md

```diff
@@ -622,6 +648,85 @@ String someDefaultValue = "someDefaultValueForTheKey";
 String value = config.getProperty(someKey, someDefaultValue);
 ```
 
+### 3.1.6 Retrieve Client Monitoring Metrics
+> Applicable to version 2.4.0 and above
+
+Apollo Client significantly enhanced observability in version 2.4.0, providing the ConfigMonitor API as well as metric export options via JMX and Prometheus. For configuration details, see [1.2.4.9 Enable Client Monitoring](#_1249-开启客户端监控).
```

Comment:
感觉in version改成since version更好一些?

---

[1.2.4.9 Enable Client Monitoring](#_1249-开启客户端监控).

这里要改成引用英文

---

See 1.2.4.9 Enable Client Monitoring for configuration details